commit 7f506c0203a59a5392fdc15a51287c6d515283fe (refs/changes/96/2996/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-11-24T11:57:06-08:00 (1 year, 10 months ago)
    
    TOOLS-1937 update sdc release documentation according to latest changes in tools (remote obsolete tooling)

diff --git a/tools/jira/LICENSE-README.md b/tools/jira/LICENSE-README.md
deleted file mode 100644
index 725e908..0000000
--- a/tools/jira/LICENSE-README.md
+++ /dev/null
@@ -1,11 +0,0 @@
-The files in the lib/ directory along with the jira.sh script in this directory
-come from the atlassian-cli-2.6.0-distribution.zip package which was available
-at the time of this writing at:
-
-  https://bobswift.atlassian.net/wiki/display/ACLI/Downloads
-
-The license files from the license/ directory of that package have been included
-here in the lib/ directory.
-
-All other scripts in this directory were created by Joyent and are subject to
-the license in the ../../LICENSE file.
diff --git a/tools/jira/README.md b/tools/jira/README.md
deleted file mode 100644
index 6270618..0000000
--- a/tools/jira/README.md
+++ /dev/null
@@ -1,42 +0,0 @@
-<!--
-    This Source Code Form is subject to the terms of the Mozilla Public
-    License, v. 2.0. If a copy of the MPL was not distributed with this
-    file, You can obtain one at http://mozilla.org/MPL/2.0/.
--->
-
-<!--
-    Copyright (c) 2014, Joyent, Inc.
--->
-
-This dir holds the scripts from/for
-<https://hub.joyent.com/wiki/display/dev/JIRA+Sprint+Release+Process>.
-
-These scripts require jira.sh (included here in the repo), see
-<https://bobswift.atlassian.net/wiki/display/JCLI/JIRA+Command+Line+Interface>.
-
-
-# Prereqs
-
-Ensure you have auth setup. You need a "~/.jiraclirc" file like this:
-
-    --user joe.blow --password password
-
-If your password has shell chars in it, tough luck.
-
-
-# Usage
-
-1. release the old version:
-
-        ./releaseversion.sh "2012-07-12 Rainier"
-
-2. add the next version (if not done earlier):
-
-        ./addsprintversion.sh "2012-07-26 Snowball-II"
-
-3. archive the old version. Note that archiving the version means that
-   devs can't remove that version from tickets they are pushing to the new
-   release now. That's a PITA, so suggest only doing this later on.
-
-        ./archiveversion.sh "2012-07-12 Rainier"
-
diff --git a/tools/jira/addsprintversion.sh b/tools/jira/addsprintversion.sh
deleted file mode 100755
index 605b46f..0000000
--- a/tools/jira/addsprintversion.sh
+++ /dev/null
@@ -1,108 +0,0 @@
-#!/bin/bash
-#
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-#
-
-#
-# Copyright (c) 2016, Joyent, Inc.
-#
-
-#
-# Update a new sprint version on Joyent Eng Jira projects.
-#
-# Usage:
-#   ./addsprintversion.sh [-c] VERSION [PROJECTS...]
-#
-# Options:
-#       -c      Continue, even if there are earlier failures. This can
-#               be useful to do a run through even if some projects already
-#               have the version.
-#
-# Example:
-#   ./addsprintversion.sh '2011-12-29 Duffman'
-#   ./addsprintversion.sh '2011-12-29 Duffman' WORKFLOW
-#   ./addsprintversion.sh -c '2011-12-29 Duffman'
-#
-
-if [[ -n "$TRACE" ]]; then
-    export PS4='[\D{%FT%TZ}] ${BASH_SOURCE}:${LINENO}: ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
-    set -o xtrace
-fi
-set -o errexit
-set -o pipefail
-
-
-TOP=$(cd $(dirname "$0") >/dev/null; pwd)
-
-
-
-# ---- mainline
-
-optContinue=
-while getopts "c" opt; do
-    case "$opt" in
-        c)
-            optContinue=true
-            ;;
-        *)
-            echo "error: unknown option $opt" >&2
-            exit 1
-            ;;
-    esac
-done
-shift $((OPTIND - 1))
-
-
-JIRACLI_OPTS="--server https://devhub.joyent.com/jira"
-JIRACLI_RC_PATH="$HOME/.jiraclirc"
-if [ ! -f "$JIRACLI_RC_PATH" ]; then
-    echo "'$JIRACLI_RC_PATH' does not exist. You need one that looks like this:"
-    echo "    --user=joe.blow --password='his-jira-password'"
-    exit 1
-fi
-JIRACLI_OPTS+=" $(cat $JIRACLI_RC_PATH)"
-
-
-if [[ -z "$1" ]]; then
-    echo "Provide VERSION name, e.g. $0 '2012-03-22 Jimbo'";
-    exit 1
-fi
-VERSION=$1
-shift
-
-YEAR=$(echo $VERSION | cut -c 1-4)
-MONTH=$(echo $VERSION | cut -c 6-7)
-DAY=$(echo $VERSION | cut -c 9-10)
-RELEASE_DATE=$MONTH/$DAY/$YEAR
-# TODO: validate this?
-
-PROJECTS="$*"
-if [[ -z "$PROJECTS" ]]; then
-    PROJECTS=$($TOP/listengprojects.sh | xargs)
-fi
-
-
-echo "This will *add* the following version to these Joyent Jira projects:"
-echo "        version: '$VERSION'"
-echo "   release date: '$RELEASE_DATE'"
-echo "       projects: $(echo "$PROJECTS" | xargs)"
-echo ""
-read -p "Hit Enter to continue..."
-echo
-
-for project in $PROJECTS
-do
-    echo "# $project: add version '$VERSION' with release date '$RELEASE_DATE'"
-    if [[ "$optContinue" == "true" ]]; then
-        # Allow failures with '-c'.
-        $TOP/jira.sh $JIRACLI_OPTS --action addVersion  \
-          --project $project --name "$VERSION" --date "$RELEASE_DATE" \
-          --dateFormat "MM/dd/yyyy" || true
-    else
-        $TOP/jira.sh $JIRACLI_OPTS --action addVersion  \
-          --project $project --name "$VERSION" --date "$RELEASE_DATE" \
-          --dateFormat "MM/dd/yyyy"
-    fi
-done
diff --git a/tools/jira/archiveversion.sh b/tools/jira/archiveversion.sh
deleted file mode 100755
index 685a64b..0000000
--- a/tools/jira/archiveversion.sh
+++ /dev/null
@@ -1,99 +0,0 @@
-#!/bin/bash
-#
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-#
-
-#
-# Copyright (c) 2016, Joyent, Inc.
-#
-
-#
-# Archive a version on Joyent Eng Jira projects.
-#
-# Usage:
-#   ./archiveversion.sh [-c] VERSION [PROJECTS...]
-#
-# Options:
-#       -c      Continue, even if there are earlier failures. This can
-#               be useful to do a run through even if some projects already
-#               have the version.
-#
-# Example:
-#   ./archiveversion.sh '2011-12-29 Duffman'
-#   ./archiveversion.sh '2011-12-29 Duffman' WORKFLOW
-#   ./archiveversion.sh -c '2011-12-29 Duffman'
-#
-
-if [[ -n "$TRACE" ]]; then
-    export PS4='[\D{%FT%TZ}] ${BASH_SOURCE}:${LINENO}: ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
-    set -o xtrace
-fi
-set -o errexit
-set -o pipefail
-
-
-TOP=$(cd $(dirname "$0") >/dev/null; pwd)
-
-
-
-# ---- mainline
-
-optContinue=
-while getopts "c" opt; do
-    case "$opt" in
-        c)
-            optContinue=true
-            ;;
-        *)
-            echo "error: unknown option $opt" >&2
-            exit 1
-            ;;
-    esac
-done
-shift $((OPTIND - 1))
-
-
-JIRACLI_OPTS="--server https://devhub.joyent.com/jira"
-JIRACLI_RC_PATH="$HOME/.jiraclirc"
-if [ ! -f "$JIRACLI_RC_PATH" ]; then
-    echo "'$JIRACLI_RC_PATH' does not exist. You need one that looks like this:"
-    echo "    --user=joe.blow --password='his-jira-password'"
-    exit 1
-fi
-JIRACLI_OPTS+=" $(cat $JIRACLI_RC_PATH)"
-
-
-if [[ -z "$1" ]]; then
-    echo "Provide VERSION name, e.g. $0 '2012-03-22 Jimbo'";
-    exit 1
-fi
-VERSION=$1
-shift
-
-PROJECTS="$*"
-if [[ -z "$PROJECTS" ]]; then
-    PROJECTS=$($TOP/listengprojects.sh | xargs)
-fi
-
-
-echo "This will *archive* the following version to these Joyent Jira projects:"
-echo "        version: '$VERSION'"
-echo "       projects: $(echo "$PROJECTS" | xargs)"
-echo ""
-read -p "Hit Enter to continue..."
-echo
-
-for project in $PROJECTS
-do
-    echo "# $project: archive version '$VERSION'"
-    if [[ "$optContinue" == "true" ]]; then
-        # Allow failures with '-c'.
-        $TOP/jira.sh $JIRACLI_OPTS --action archiveVersion  \
-          --project $project --name "$VERSION" || true
-    else
-        $TOP/jira.sh $JIRACLI_OPTS --action archiveVersion  \
-          --project $project --name "$VERSION"
-    fi
-done
diff --git a/tools/jira/jira.sh b/tools/jira/jira.sh
deleted file mode 100755
index 8410a6f..0000000
--- a/tools/jira/jira.sh
+++ /dev/null
@@ -1,17 +0,0 @@
-#!/bin/bash
-#
-# LICENSE: See lib/LICENSE.txt
-#
-
-if [[ -n "$TRACE" ]]; then
-    export PS4='[\D{%FT%TZ}] ${BASH_SOURCE}:${LINENO}: ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
-    set -o xtrace
-fi
-set -o errexit
-set -o pipefail
-
-# Comments
-# - Customize for your installation, for instance you might want to add default parameters like the following:
-# java -jar `dirname $0`/lib/jira-cli-2.6.0.jar --server http://my-server --user automation --password automation "$@"
-
-java -jar `dirname $0`/lib/jira-cli-2.6.0.jar "$@"
diff --git a/tools/jira/lib/LICENSE-APACHE.txt b/tools/jira/lib/LICENSE-APACHE.txt
deleted file mode 100644
index d645695..0000000
--- a/tools/jira/lib/LICENSE-APACHE.txt
+++ /dev/null
@@ -1,202 +0,0 @@
-
-                                 Apache License
-                           Version 2.0, January 2004
-                        http://www.apache.org/licenses/
-
-   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
-
-   1. Definitions.
-
-      "License" shall mean the terms and conditions for use, reproduction,
-      and distribution as defined by Sections 1 through 9 of this document.
-
-      "Licensor" shall mean the copyright owner or entity authorized by
-      the copyright owner that is granting the License.
-
-      "Legal Entity" shall mean the union of the acting entity and all
-      other entities that control, are controlled by, or are under common
-      control with that entity. For the purposes of this definition,
-      "control" means (i) the power, direct or indirect, to cause the
-      direction or management of such entity, whether by contract or
-      otherwise, or (ii) ownership of fifty percent (50%) or more of the
-      outstanding shares, or (iii) beneficial ownership of such entity.
-
-      "You" (or "Your") shall mean an individual or Legal Entity
-      exercising permissions granted by this License.
-
-      "Source" form shall mean the preferred form for making modifications,
-      including but not limited to software source code, documentation
-      source, and configuration files.
-
-      "Object" form shall mean any form resulting from mechanical
-      transformation or translation of a Source form, including but
-      not limited to compiled object code, generated documentation,
-      and conversions to other media types.
-
-      "Work" shall mean the work of authorship, whether in Source or
-      Object form, made available under the License, as indicated by a
-      copyright notice that is included in or attached to the work
-      (an example is provided in the Appendix below).
-
-      "Derivative Works" shall mean any work, whether in Source or Object
-      form, that is based on (or derived from) the Work and for which the
-      editorial revisions, annotations, elaborations, or other modifications
-      represent, as a whole, an original work of authorship. For the purposes
-      of this License, Derivative Works shall not include works that remain
-      separable from, or merely link (or bind by name) to the interfaces of,
-      the Work and Derivative Works thereof.
-
-      "Contribution" shall mean any work of authorship, including
-      the original version of the Work and any modifications or additions
-      to that Work or Derivative Works thereof, that is intentionally
-      submitted to Licensor for inclusion in the Work by the copyright owner
-      or by an individual or Legal Entity authorized to submit on behalf of
-      the copyright owner. For the purposes of this definition, "submitted"
-      means any form of electronic, verbal, or written communication sent
-      to the Licensor or its representatives, including but not limited to
-      communication on electronic mailing lists, source code control systems,
-      and issue tracking systems that are managed by, or on behalf of, the
-      Licensor for the purpose of discussing and improving the Work, but
-      excluding communication that is conspicuously marked or otherwise
-      designated in writing by the copyright owner as "Not a Contribution."
-
-      "Contributor" shall mean Licensor and any individual or Legal Entity
-      on behalf of whom a Contribution has been received by Licensor and
-      subsequently incorporated within the Work.
-
-   2. Grant of Copyright License. Subject to the terms and conditions of
-      this License, each Contributor hereby grants to You a perpetual,
-      worldwide, non-exclusive, no-charge, royalty-free, irrevocable
-      copyright license to reproduce, prepare Derivative Works of,
-      publicly display, publicly perform, sublicense, and distribute the
-      Work and such Derivative Works in Source or Object form.
-
-   3. Grant of Patent License. Subject to the terms and conditions of
-      this License, each Contributor hereby grants to You a perpetual,
-      worldwide, non-exclusive, no-charge, royalty-free, irrevocable
-      (except as stated in this section) patent license to make, have made,
-      use, offer to sell, sell, import, and otherwise transfer the Work,
-      where such license applies only to those patent claims licensable
-      by such Contributor that are necessarily infringed by their
-      Contribution(s) alone or by combination of their Contribution(s)
-      with the Work to which such Contribution(s) was submitted. If You
-      institute patent litigation against any entity (including a
-      cross-claim or counterclaim in a lawsuit) alleging that the Work
-      or a Contribution incorporated within the Work constitutes direct
-      or contributory patent infringement, then any patent licenses
-      granted to You under this License for that Work shall terminate
-      as of the date such litigation is filed.
-
-   4. Redistribution. You may reproduce and distribute copies of the
-      Work or Derivative Works thereof in any medium, with or without
-      modifications, and in Source or Object form, provided that You
-      meet the following conditions:
-
-      (a) You must give any other recipients of the Work or
-          Derivative Works a copy of this License; and
-
-      (b) You must cause any modified files to carry prominent notices
-          stating that You changed the files; and
-
-      (c) You must retain, in the Source form of any Derivative Works
-          that You distribute, all copyright, patent, trademark, and
-          attribution notices from the Source form of the Work,
-          excluding those notices that do not pertain to any part of
-          the Derivative Works; and
-
-      (d) If the Work includes a "NOTICE" text file as part of its
-          distribution, then any Derivative Works that You distribute must
-          include a readable copy of the attribution notices contained
-          within such NOTICE file, excluding those notices that do not
-          pertain to any part of the Derivative Works, in at least one
-          of the following places: within a NOTICE text file distributed
-          as part of the Derivative Works; within the Source form or
-          documentation, if provided along with the Derivative Works; or,
-          within a display generated by the Derivative Works, if and
-          wherever such third-party notices normally appear. The contents
-          of the NOTICE file are for informational purposes only and
-          do not modify the License. You may add Your own attribution
-          notices within Derivative Works that You distribute, alongside
-          or as an addendum to the NOTICE text from the Work, provided
-          that such additional attribution notices cannot be construed
-          as modifying the License.
-
-      You may add Your own copyright statement to Your modifications and
-      may provide additional or different license terms and conditions
-      for use, reproduction, or distribution of Your modifications, or
-      for any such Derivative Works as a whole, provided Your use,
-      reproduction, and distribution of the Work otherwise complies with
-      the conditions stated in this License.
-
-   5. Submission of Contributions. Unless You explicitly state otherwise,
-      any Contribution intentionally submitted for inclusion in the Work
-      by You to the Licensor shall be under the terms and conditions of
-      this License, without any additional terms or conditions.
-      Notwithstanding the above, nothing herein shall supersede or modify
-      the terms of any separate license agreement you may have executed
-      with Licensor regarding such Contributions.
-
-   6. Trademarks. This License does not grant permission to use the trade
-      names, trademarks, service marks, or product names of the Licensor,
-      except as required for reasonable and customary use in describing the
-      origin of the Work and reproducing the content of the NOTICE file.
-
-   7. Disclaimer of Warranty. Unless required by applicable law or
-      agreed to in writing, Licensor provides the Work (and each
-      Contributor provides its Contributions) on an "AS IS" BASIS,
-      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
-      implied, including, without limitation, any warranties or conditions
-      of TITLE, NON-INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR A
-      PARTICULAR PURPOSE. You are solely responsible for determining the
-      appropriateness of using or redistributing the Work and assume any
-      risks associated with Your exercise of permissions under this License.
-
-   8. Limitation of Liability. In no event and under no legal theory,
-      whether in tort (including negligence), contract, or otherwise,
-      unless required by applicable law (such as deliberate and grossly
-      negligent acts) or agreed to in writing, shall any Contributor be
-      liable to You for damages, including any direct, indirect, special,
-      incidental, or consequential damages of any character arising as a
-      result of this License or out of the use or inability to use the
-      Work (including but not limited to damages for loss of goodwill,
-      work stoppage, computer failure or malfunction, or any and all
-      other commercial damages or losses), even if such Contributor
-      has been advised of the possibility of such damages.
-
-   9. Accepting Warranty or Additional Liability. While redistributing
-      the Work or Derivative Works thereof, You may choose to offer,
-      and charge a fee for, acceptance of support, warranty, indemnity,
-      or other liability obligations and/or rights consistent with this
-      License. However, in accepting such obligations, You may act only
-      on Your own behalf and on Your sole responsibility, not on behalf
-      of any other Contributor, and only if You agree to indemnify,
-      defend, and hold each Contributor harmless for any liability
-      incurred by, or claims asserted against, such Contributor by reason
-      of your accepting any such warranty or additional liability.
-
-   END OF TERMS AND CONDITIONS
-
-   APPENDIX: How to apply the Apache License to your work.
-
-      To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "[]"
-      replaced with your own identifying information. (Don't include
-      the brackets!)  The text should be enclosed in the appropriate
-      comment syntax for the file format. We also recommend that a
-      file or class name and description of purpose be included on the
-      same "printed page" as the copyright notice for easier
-      identification within third-party archives.
-
-   Copyright [yyyy] [name of copyright owner]
-
-   Licensed under the Apache License, Version 2.0 (the "License");
-   you may not use this file except in compliance with the License.
-   You may obtain a copy of the License at
-
-       http://www.apache.org/licenses/LICENSE-2.0
-
-   Unless required by applicable law or agreed to in writing, software
-   distributed under the License is distributed on an "AS IS" BASIS,
-   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-   See the License for the specific language governing permissions and
-   limitations under the License.
diff --git a/tools/jira/lib/LICENSE-DOLBY.txt b/tools/jira/lib/LICENSE-DOLBY.txt
deleted file mode 100644
index 2739b7f..0000000
--- a/tools/jira/lib/LICENSE-DOLBY.txt
+++ /dev/null
@@ -1,21 +0,0 @@
-Copyright (c) 2007, Copyright (c) 2007, Dolby Laboratories, Inc.
-
-All rights reserved.
-
-Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
-
-    * Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
-    * Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
-    * Neither the name of the Dolby Laboratories, Inc. nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
-
-THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
-"AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
-LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
-A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
-CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
-EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
-PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
-PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
-LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
-NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
-SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
diff --git a/tools/jira/lib/LICENSE-JSAP.txt b/tools/jira/lib/LICENSE-JSAP.txt
deleted file mode 100644
index 7992529..0000000
--- a/tools/jira/lib/LICENSE-JSAP.txt
+++ /dev/null
@@ -1,31 +0,0 @@
-JSAP - Java Simple Argument Parser
-Copyright (c) 2002-2004, Martian Software, Inc.
-
-
-LICENSE
--------  
-JSAP is licensed under the Lesser GNU Public License.
-A copy of this license is available at
-http://www.fsf.org/licenses/licenses.html#LGPL
-
-Alternate licensing terms may be obtained by contacting
-the author: http://www.martiansoftware.com/contact.html
-
-
-JAVA LGPL CLARIFICATION
------------------------
-JSAP is Free Software.  The LGPL license is sufficiently
-flexible to allow the use of JSAP in both open source
-and commercial projects.  Using JSAP (by importing JSAP's
-public interfaces in your Java code), and extending JSAP
-(by subclassing) are considered by the authors of JSAP
-to be dynamic linking.  Hence our interpretation of the
-LGPL is that the use of the unmodified JSAP source or
-binary, or the rebundling of unmodified JSAP classes into
-your program's .jar file, does not affect the license of 
-your application code.
-
-If you modify JSAP and redistribute your modifications,
-the LGPL applies. 
-
-(based upon similar clarification at http://www.hibernate.org/ )
diff --git a/tools/jira/lib/LICENSE-POSTGRESQL.txt b/tools/jira/lib/LICENSE-POSTGRESQL.txt
deleted file mode 100644
index bd8ecce..0000000
--- a/tools/jira/lib/LICENSE-POSTGRESQL.txt
+++ /dev/null
@@ -1,30 +0,0 @@
-BSD License
-
-The PostgreSQL JDBC driver is distributed under the BSD license, same as the server. The simplest explanation of the licensing terms is that you can do whatever you want with the product and source code as long as you don't claim you wrote it or sue us. You should give it a read though, it's only half a page.
-
-Copyright (c) 1997-2010, PostgreSQL Global Development Group
-All rights reserved.
-
-Redistribution and use in source and binary forms, with or without
-modification, are permitted provided that the following conditions are met:
-
-1. Redistributions of source code must retain the above copyright notice,
-   this list of conditions and the following disclaimer.
-2. Redistributions in binary form must reproduce the above copyright notice,
-   this list of conditions and the following disclaimer in the documentation
-   and/or other materials provided with the distribution.
-3. Neither the name of the PostgreSQL Global Development Group nor the names
-   of its contributors may be used to endorse or promote products derived
-   from this software without specific prior written permission.
-
-THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
-AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
-IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
-ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
-LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
-CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
-SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
-INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
-CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
-ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
-POSSIBILITY OF SUCH DAMAGE.
diff --git a/tools/jira/lib/LICENSE.txt b/tools/jira/lib/LICENSE.txt
deleted file mode 100644
index 28efe28..0000000
--- a/tools/jira/lib/LICENSE.txt
+++ /dev/null
@@ -1,25 +0,0 @@
-Copyright (c) 2005, 2009 Bob Swift and other contributors.
-All rights reserved.
-
-Redistribution and use in source and binary forms, with or without
-modification, are permitted provided that the following conditions are met:
-
-    * Redistributions of source code must retain the above copyright notice,
-      this list of conditions and the following disclaimer.
-    * Redistributions in binary form must reproduce the above copyright notice,
-      this list of conditions and the following disclaimer in the documentation
-      and/or other materials provided with the distribution.
-    * The names of contributors may not
-      be used to endorse or promote products derived from this software without
-      specific prior written permission.
-
-THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
-ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
-WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
-DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
-ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
-(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
-LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
-ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
-(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
-SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
\ No newline at end of file
diff --git a/tools/jira/lib/NOTICES.txt b/tools/jira/lib/NOTICES.txt
deleted file mode 100644
index be26de0..0000000
--- a/tools/jira/lib/NOTICES.txt
+++ /dev/null
@@ -1,20 +0,0 @@
-
-   For license information, see LICENSE files included with
-   this distribution as well as LICENSE files packaged in
-   the various jar files included with this distribution.
-
-   This distribution includes binary distributions from
-   other providers including, but not limited to, the following:
-
-   This product includes software developed by
-   The Apache Software Foundation (http://www.apache.org/).
-
-   This product includes software developed by
-   Martian Software, Inc. (http://www.martiansoftware.com).
-
-   This product includes software developed by
-   Dolby Laboratories, Inc. (http://www.dolby.co.uk).
-
-
-   This product includes software developed by
-   PostgreSQL Global Development Group. (http://jdbc.postgresql.org).
diff --git a/tools/jira/lib/activation-1.1.jar b/tools/jira/lib/activation-1.1.jar
deleted file mode 100644
index 53f82a1..0000000
Binary files a/tools/jira/lib/activation-1.1.jar and /dev/null differ
diff --git a/tools/jira/lib/avalon-framework-4.1.3.jar b/tools/jira/lib/avalon-framework-4.1.3.jar
deleted file mode 100644
index fd72580..0000000
Binary files a/tools/jira/lib/avalon-framework-4.1.3.jar and /dev/null differ
diff --git a/tools/jira/lib/axis-1.4.jar b/tools/jira/lib/axis-1.4.jar
deleted file mode 100644
index 20b09a5..0000000
Binary files a/tools/jira/lib/axis-1.4.jar and /dev/null differ
diff --git a/tools/jira/lib/axis-jaxrpc-1.4.jar b/tools/jira/lib/axis-jaxrpc-1.4.jar
deleted file mode 100644
index a2c13d9..0000000
Binary files a/tools/jira/lib/axis-jaxrpc-1.4.jar and /dev/null differ
diff --git a/tools/jira/lib/axis-saaj-1.4.jar b/tools/jira/lib/axis-saaj-1.4.jar
deleted file mode 100644
index 4ea696e..0000000
Binary files a/tools/jira/lib/axis-saaj-1.4.jar and /dev/null differ
diff --git a/tools/jira/lib/axis-wsdl4j-1.5.1.jar b/tools/jira/lib/axis-wsdl4j-1.5.1.jar
deleted file mode 100644
index c6254ee..0000000
Binary files a/tools/jira/lib/axis-wsdl4j-1.5.1.jar and /dev/null differ
diff --git a/tools/jira/lib/bamboo-cli-2.6.0.jar b/tools/jira/lib/bamboo-cli-2.6.0.jar
deleted file mode 100644
index ff62f13..0000000
Binary files a/tools/jira/lib/bamboo-cli-2.6.0.jar and /dev/null differ
diff --git a/tools/jira/lib/cli-2.6.0.jar b/tools/jira/lib/cli-2.6.0.jar
deleted file mode 100644
index ae1ed06..0000000
Binary files a/tools/jira/lib/cli-2.6.0.jar and /dev/null differ
diff --git a/tools/jira/lib/cli-soap-2.6.0.jar b/tools/jira/lib/cli-soap-2.6.0.jar
deleted file mode 100644
index 1c52771..0000000
Binary files a/tools/jira/lib/cli-soap-2.6.0.jar and /dev/null differ
diff --git a/tools/jira/lib/commons-discovery-0.2.jar b/tools/jira/lib/commons-discovery-0.2.jar
deleted file mode 100644
index b885548..0000000
Binary files a/tools/jira/lib/commons-discovery-0.2.jar and /dev/null differ
diff --git a/tools/jira/lib/commons-lang-2.4.jar b/tools/jira/lib/commons-lang-2.4.jar
deleted file mode 100644
index 532939e..0000000
Binary files a/tools/jira/lib/commons-lang-2.4.jar and /dev/null differ
diff --git a/tools/jira/lib/commons-logging-1.1.jar b/tools/jira/lib/commons-logging-1.1.jar
deleted file mode 100644
index 2ff9bbd..0000000
Binary files a/tools/jira/lib/commons-logging-1.1.jar and /dev/null differ
diff --git a/tools/jira/lib/confluence-cli-2.6.0.jar b/tools/jira/lib/confluence-cli-2.6.0.jar
deleted file mode 100644
index cdec5cc..0000000
Binary files a/tools/jira/lib/confluence-cli-2.6.0.jar and /dev/null differ
diff --git a/tools/jira/lib/confluence-soap-4.0.0.jar b/tools/jira/lib/confluence-soap-4.0.0.jar
deleted file mode 100644
index c8dcebe..0000000
Binary files a/tools/jira/lib/confluence-soap-4.0.0.jar and /dev/null differ
diff --git a/tools/jira/lib/crucible-cli-2.6.0.jar b/tools/jira/lib/crucible-cli-2.6.0.jar
deleted file mode 100644
index 107a404..0000000
Binary files a/tools/jira/lib/crucible-cli-2.6.0.jar and /dev/null differ
diff --git a/tools/jira/lib/fisheye-cli-2.6.0.jar b/tools/jira/lib/fisheye-cli-2.6.0.jar
deleted file mode 100644
index 2b0ff8a..0000000
Binary files a/tools/jira/lib/fisheye-cli-2.6.0.jar and /dev/null differ
diff --git a/tools/jira/lib/jdbc/postgresql-8.4-702.jdbc3.jar b/tools/jira/lib/jdbc/postgresql-8.4-702.jdbc3.jar
deleted file mode 100644
index bf01aee..0000000
Binary files a/tools/jira/lib/jdbc/postgresql-8.4-702.jdbc3.jar and /dev/null differ
diff --git a/tools/jira/lib/jira-cli-2.6.0.jar b/tools/jira/lib/jira-cli-2.6.0.jar
deleted file mode 100644
index 49df83a..0000000
Binary files a/tools/jira/lib/jira-cli-2.6.0.jar and /dev/null differ
diff --git a/tools/jira/lib/jira-soap-4.3.2.jar b/tools/jira/lib/jira-soap-4.3.2.jar
deleted file mode 100644
index f9e7319..0000000
Binary files a/tools/jira/lib/jira-soap-4.3.2.jar and /dev/null differ
diff --git a/tools/jira/lib/jsap-2.1.jar b/tools/jira/lib/jsap-2.1.jar
deleted file mode 100644
index 15d1f37..0000000
Binary files a/tools/jira/lib/jsap-2.1.jar and /dev/null differ
diff --git a/tools/jira/lib/json-simple-1.1.jar b/tools/jira/lib/json-simple-1.1.jar
deleted file mode 100644
index f395f41..0000000
Binary files a/tools/jira/lib/json-simple-1.1.jar and /dev/null differ
diff --git a/tools/jira/lib/log4j-1.2.12.jar b/tools/jira/lib/log4j-1.2.12.jar
deleted file mode 100644
index 9b5a720..0000000
Binary files a/tools/jira/lib/log4j-1.2.12.jar and /dev/null differ
diff --git a/tools/jira/lib/logkit-1.0.1.jar b/tools/jira/lib/logkit-1.0.1.jar
deleted file mode 100644
index d3250ee..0000000
Binary files a/tools/jira/lib/logkit-1.0.1.jar and /dev/null differ
diff --git a/tools/jira/lib/mail-1.4.jar b/tools/jira/lib/mail-1.4.jar
deleted file mode 100644
index fd4555b..0000000
Binary files a/tools/jira/lib/mail-1.4.jar and /dev/null differ
diff --git a/tools/jira/lib/servlet-api-2.3.jar b/tools/jira/lib/servlet-api-2.3.jar
deleted file mode 100644
index cd5b351..0000000
Binary files a/tools/jira/lib/servlet-api-2.3.jar and /dev/null differ
diff --git a/tools/jira/releaseversion.sh b/tools/jira/releaseversion.sh
deleted file mode 100755
index e9fd2c1..0000000
--- a/tools/jira/releaseversion.sh
+++ /dev/null
@@ -1,99 +0,0 @@
-#!/bin/bash
-#
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-#
-
-#
-# Copyright (c) 2016, Joyent, Inc.
-#
-
-#
-# Release a version on Joyent Eng Jira projects.
-#
-# Usage:
-#   ./releaseversion.sh [-c] VERSION [PROJECTS...]
-#
-# Options:
-#       -c      Continue, even if there are earlier failures. This can
-#               be useful to do a run through even if some projects already
-#               have the version.
-#
-# Example:
-#   ./releaseversion.sh '2011-12-29 Duffman'
-#   ./releaseversion.sh '2011-12-29 Duffman' WORKFLOW
-#   ./releaseversion.sh -c '2011-12-29 Duffman'
-#
-
-if [[ -n "$TRACE" ]]; then
-    export PS4='[\D{%FT%TZ}] ${BASH_SOURCE}:${LINENO}: ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
-    set -o xtrace
-fi
-set -o errexit
-set -o pipefail
-
-
-TOP=$(cd $(dirname "$0") >/dev/null; pwd)
-
-
-
-# ---- mainline
-
-optContinue=
-while getopts "c" opt; do
-    case "$opt" in
-        c)
-            optContinue=true
-            ;;
-        *)
-            echo "error: unknown option $opt" >&2
-            exit 1
-            ;;
-    esac
-done
-shift $((OPTIND - 1))
-
-
-JIRACLI_OPTS="--server https://devhub.joyent.com/jira"
-JIRACLI_RC_PATH="$HOME/.jiraclirc"
-if [ ! -f "$JIRACLI_RC_PATH" ]; then
-    echo "'$JIRACLI_RC_PATH' does not exist. You need one that looks like this:"
-    echo "    --user=joe.blow --password='his-jira-password'"
-    exit 1
-fi
-JIRACLI_OPTS+=" $(cat $JIRACLI_RC_PATH)"
-
-
-if [[ -z "$1" ]]; then
-    echo "Provide VERSION name, e.g. $0 '2012-03-22 Jimbo'";
-    exit 1
-fi
-VERSION=$1
-shift
-
-PROJECTS="$*"
-if [[ -z "$PROJECTS" ]]; then
-    PROJECTS=$($TOP/listengprojects.sh | xargs)
-fi
-
-
-echo "This will *release* the following version to these Joyent Jira projects:"
-echo "        version: '$VERSION'"
-echo "       projects: $(echo "$PROJECTS" | xargs)"
-echo ""
-read -p "Hit Enter to continue..."
-echo
-
-for project in $PROJECTS
-do
-    echo "# $project: release version '$VERSION'"
-    if [[ "$optContinue" == "true" ]]; then
-        # Allow failures with '-c'.
-        $TOP/jira.sh $JIRACLI_OPTS --action releaseVersion  \
-          --project $project --name "$VERSION" || true
-    else
-        $TOP/jira.sh $JIRACLI_OPTS --action releaseVersion  \
-          --project $project --name "$VERSION"
-    fi
-done
diff --git a/tools/jira/unreleaseversion.sh b/tools/jira/unreleaseversion.sh
deleted file mode 100755
index eb357b9..0000000
--- a/tools/jira/unreleaseversion.sh
+++ /dev/null
@@ -1,99 +0,0 @@
-#!/bin/bash
-#
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-#
-
-#
-# Copyright (c) 2016, Joyent, Inc.
-#
-
-#
-# *Un*release a version on Joyent Eng Jira projects.
-#
-# Usage:
-#   ./unreleaseversion.sh [-c] VERSION [PROJECTS...]
-#
-# Options:
-#       -c      Continue, even if there are earlier failures. This can
-#               be useful to do a run through even if some projects already
-#               have the version.
-#
-# Example:
-#   ./unreleaseversion.sh '2011-12-29 Duffman'
-#   ./unreleaseversion.sh '2011-12-29 Duffman' WORKFLOW
-#   ./unreleaseversion.sh -c '2011-12-29 Duffman'
-#
-
-if [[ -n "$TRACE" ]]; then
-    export PS4='[\D{%FT%TZ}] ${BASH_SOURCE}:${LINENO}: ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
-    set -o xtrace
-fi
-set -o errexit
-set -o pipefail
-
-
-TOP=$(cd $(dirname "$0") >/dev/null; pwd)
-
-
-
-# ---- mainline
-
-optContinue=
-while getopts "c" opt; do
-    case "$opt" in
-        c)
-            optContinue=true
-            ;;
-        *)
-            echo "error: unknown option $opt" >&2
-            exit 1
-            ;;
-    esac
-done
-shift $((OPTIND - 1))
-
-
-JIRACLI_OPTS="--server https://devhub.joyent.com/jira"
-JIRACLI_RC_PATH="$HOME/.jiraclirc"
-if [ ! -f "$JIRACLI_RC_PATH" ]; then
-    echo "'$JIRACLI_RC_PATH' does not exist. You need one that looks like this:"
-    echo "    --user=joe.blow --password='his-jira-password'"
-    exit 1
-fi
-JIRACLI_OPTS+=" $(cat $JIRACLI_RC_PATH)"
-
-
-if [[ -z "$1" ]]; then
-    echo "Provide VERSION name, e.g. $0 '2012-03-22 Jimbo'";
-    exit 1
-fi
-VERSION=$1
-shift
-
-PROJECTS="$*"
-if [[ -z "$PROJECTS" ]]; then
-    PROJECTS=$($TOP/listengprojects.sh | xargs)
-fi
-
-
-echo "This will *unrelease* the following version to these Joyent Jira projects:"
-echo "        version: '$VERSION'"
-echo "       projects: $(echo "$PROJECTS" | xargs)"
-echo ""
-read -p "Hit Enter to continue..."
-echo
-
-for project in $PROJECTS
-do
-    echo "# $project: unrelease version '$VERSION'"
-    if [[ "$optContinue" == "true" ]]; then
-        # Allow failures with '-c'.
-        $TOP/jira.sh $JIRACLI_OPTS --action unreleaseVersion  \
-          --project $project --name "$VERSION" || true
-    else
-        $TOP/jira.sh $JIRACLI_OPTS --action unreleaseVersion  \
-          --project $project --name "$VERSION"
-    fi
-done
